### PR TITLE
[wasm] eqz, copysign, trunc, and nearest

### PR DESCRIPTION
**Pull request by commonlisp**
This PR covers interpreter & codegen for eqz and copysign and interpreter support for trunc and nearest with codegen forthcoming.

### CODE REVIEW

File: lib/Runtime/ByteCode/OpCodes.h [hunk 1]
```diff
@@ -743,6 +742,11 @@ MACRO_BACKEND_ONLY(     SlotArrayCheck,     Empty,          OpCanCSE)
 MACRO_BACKEND_ONLY(     FrameDisplayCheck,  Empty,          OpCanCSE)
 MACRO_EXTEND(           BeginBodyScope,     Empty,          OpSideEffect)
 
+MACRO_BACKEND_ONLY(Copysign_A, Empty, OpInlinableBuiltIn | OpTempNumberSources | OpCanCSE | OpProducesNumber)
+MACRO_BACKEND_ONLY(Trunc_A,    Empty, OpInlinableBuiltIn | OpTempNumberSources | OpCanCSE | OpProducesNumber)
+MACRO_BACKEND_ONLY(Nearest_A,  Empty, OpInlinableBuiltIn | OpTempNumberSources | OpCanCSE | OpProducesNumber)
```
Comment (MikeHolman) [inline lib/Runtime/ByteCode/OpCodes.h]: I don't believe we need the `OpInlinableBuiltIn` flag

**Comment by commonlisp** [comment]
@dotnet-bot please test this

File: bin/ch/ch.vcxproj [hunk 2]
```diff
@@ -27,6 +27,7 @@
         $(IntDir);
         %(AdditionalIncludeDirectories);
       </AdditionalIncludeDirectories>
+      <PreprocessToFile Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">false</PreprocessToFile>
```
Comment (MikeHolman) [inline bin/ch/ch.vcxproj]: what is this doing?

**Comment by MikeHolman** [comment]
lgtm

File: bin/ch/ch.vcxproj (hunk 2 above)
Comment (commonlisp) [inline bin/ch/ch.vcxproj]: Yeah, good catch. This was left over when I was hunting down extended prefix exhaustion.

**Pull request merged**